docs/examples/use_cases/tensorflow/resnet-n/nvutils/image_processing.py

```diff
@@ -217,6 +217,9 @@ def __init__(self,
     device_id = hvd.local_rank()
     shard_id = hvd.rank()
     num_gpus = hvd.size()
+    kwargs = dict()
+    if deterministic:
+       kwargs['seed'] = 7 * (1 + hvd.rank())
```

Comment:
Correct me if I'm wrong, but If `deterministic == False`, `kwargs['seed']` wouldn't still be None?
Shouldn't it be like:
```python
kwargs['seed']=7 * (1 + hvd.rank()) if deterministic else -1
```

Reply:
If deterministic is false then there is no such keyword arg and the default DALI value use used.